### ISSUE TITLE
ANAN-10E power 

### ISSUE THREAD

**Issue by k1yw** [g0orx/linhpsdr#4]
I'm unable to set TX power to more than 1W. Tried setting Tired Tune Percent to 100, also normal TX using FM mode, Set PA calibration to 100, still no more than 1W out.

**Comment by g0orx** [comment]
To increase output power you need to lower the value in the PA calibration.  Let me know if that solves the problem.

**Comment by k1yw** [comment]
No, it appears that changing the PA calibration value does not change the power output at all. Trying this in FM mode, drive is at 100%, cannot achieve more than 1W output .  I can reduce the power to less than 1W by lowering the TX drive of course.

**Comment by nubbyless** [comment]
I have the same exact issue with anan10e also same issues when I tried pihpsdr pa adjustments seem to do nothing on any band for me

**Comment by k1yw** [comment]
I believe John fixed this in the latest update.

**Comment by nubbyless** [comment]
hi how can i get the latest update i installed yesterday by cloning git
